src/wallet/wallet2.cpp

```diff
@@ -7306,6 +7307,7 @@ wallet2::stake_result wallet2::create_stake_tx(const crypto::public_key& service
     return result;
   }
 
+  assert(result.status != stake_result_status::invalid);
```

Comment:
Are we relying on result to be default initialized to stake_result_status::invalid? How about we make it explicit?

Reply:
Sure. Fixed